server/gaps/gaps_insert_functions.py

```diff
@@ -0,0 +1,167 @@
+
+from helpers import cold_storage_type_id_lookup
+
+
+def changelog_stmt(record_id, table_name):
+    # Thank you to whoever set the default value of cursor in ChangeLog to properly generate the next value!
+    # nextval('changelog_cursor_seq'::regclass)
+    return f"INSERT INTO changelog (record_id, table_name, row_action) VALUES ('{record_id}', '{table_name}', 'UPSERT');"
+
+
+def item_link_stmt(item_id):
+    return f"INSERT INTO item_link (id, item_id) VALUES ('{item_id}', '{item_id}') ON CONFLICT DO NOTHING;"
+
+def upsert_item_stmt(item_id, row):
+    item_name = row['VaccineType']
+    item_code = row['VaccineTypeName']
```

Comment:
Typename might need to be included in item name, getting this with covid vaccines and rotavirus (codes are all different but this looks not great...) 
![Screenshot 2024-11-19 at 11 00 46 AM](https://github.com/user-attachments/assets/9b33eb10-ecad-43ba-9399-30e5b45dd05a)